db/seed.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require('mongoose');
+const Listing = require('./Listing.js');
+const faker = require('faker');
+
+const sampleListings = [];
+
+let genBookedDates = function() {
+  const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+  let dates = {};
+  
+  for (let i = 0; i < 100; i++) {
+    let date = faker.date.future(2)
+    let year = date.getFullYear();
+    let monthNumber = date.getMonth();
+    let month = months[monthNumber];
+    let day = date.getDate();
+
+    if (object[year] === undefined) {
+      object[year] = {};
+      object[year][month] = [];
+    } else if (object[year][month] === undefined) {
+      object[year][month] = [];
+    }
+    if (object[year][month].indexOf(day) === -1) {
+      object[year][month].push(day);
+    }
+  }
+  return dates;
+}
+
+for (let i = 1; i <= 100; i++) {
+  listing = new Listing({
```

Comment:
After reading the documentation I don't believe that you need to create a new instance of Listing each time to create a new document since you are using the insertMany method.

https://mongoosejs.com/docs/api.html#model_Model.insertMany

insertMany will automatically create those documents for you without you having to use 'new Listing' to create a document to begin with.

Reply:
Thank you! I updated this.